src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DynamicFiles/RazorDynamicFileInfoProvider.cs

```diff
@@ -73,7 +73,7 @@ public void UpdateLSPFileInfo(Uri documentUri, IDynamicDocumentContainer documen
         documentContainer.SetSupportsDiagnostics(true);
 
         // TODO: This needs to use the project key somehow, rather than assuming all generated content is the same
-        var filePath = FilePathService.GetProjectSystemFilePath(documentUri);
+        var filePath = AbstractFilePathService.GetProjectSystemFilePath(documentUri);
```

Comment:
I did a FAR on this method and it seems to only be used in VS? The comment in the method would indicate that it was useful for differences between VS and VS Mac. If that's the case, consider moving it someplace in the VS layer.